tests/ApiTest.php

```diff
@@ -121,7 +124,6 @@ public function testCreateCallWithAmdAndGetCallState() {
         //get phone call information
         $response = $this->bandwidthClient->getVoice()->getClient()->getCall(getenv("BW_ACCOUNT_ID"), $callId);
         $this->assertTrue(strlen($response->getResult()->state) > 0);
-        $this->assertTrue(is_a($response->getResult()->enqueuedTime, 'DateTime'));
```

Comment:
I think you can leave this assert since `enqueuedTime` is always present in the `getCall` response